src/lake/Lake/CLI/Init.lean

```diff
@@ -267,6 +269,11 @@ def initPkg (dir : FilePath) (name : String) (tmp : InitTemplate) (lang : Config
     else
       logWarning "failed to initialize git repository"
 
+  -- Initialize a README.md file if none exists.
+  let readmeFile := dir / "README.md"
+  unless (← readmeFile.pathExists) do
+    IO.FS.writeFile readmeFile (readmeFileContents pkgName.toString)
```

Comment:
```suggestion
    IO.FS.writeFile readmeFile (readmeFileContents name)
```

FYI, this should use  `name` (the `String` name the user entered) instead of `pkgName.toString` (the potentially escaped Lean identifier).